scripts/increment_auto_off.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
```

Comment:
In the last few commits I've entirely reworked this file

Calling attention to it because the whole thing is green either way

Reply:
This is even more true now